node/metrics/src/names.rs

```diff
@@ -44,8 +44,15 @@ pub(super) const GAUGE_NAMES: [&str; 26] = [
     tcp::TCP_TASKS,
 ];
 
-pub(super) const HISTOGRAM_NAMES: [&str; 3] =
-    [bft::COMMIT_ROUNDS_LATENCY, consensus::CERTIFICATE_COMMIT_LATENCY, consensus::BLOCK_LATENCY];
+pub(super) const HISTOGRAM_NAMES: [&str; 7] = [
+    bft::COMMIT_ROUNDS_LATENCY,
+    consensus::CERTIFICATE_COMMIT_LATENCY,
+    consensus::BLOCK_LATENCY,
+    tcp::TCP_GATEWAY_EVENTS_OUTBOUND,
+    tcp::TCP_GATEWAY_EVENTS_INBOUND,
```

Comment:
```suggestion
    tcp::TCP_GATEWAY_BYTES_OUTBOUND,
    tcp::TCP_GATEWAY_BYTES_INBOUND,
```
IMO events would be more apt for the number of sent/received messages (which we might also want to track; they could even be used to detect DoS attempts)